web/src/agents/node/get_last_sequence.rs

```diff
@@ -0,0 +1,41 @@
+use at2_node::client::{self, Client};
+use at2_ns::ThinUser;
+use wasm_bindgen_futures::spawn_local;
+use yew::worker::*;
+
+use crate::config::Config;
+
+/// Get the last sequence for a user
```

Comment:
Sequence of what? Transactions?

Reply:
every message send on the network is a tuple of the sender, it's counter/sequence number, and a payload. this agent retrieve the latest processed sequence number for a given user, to show the number of confirmed transactions during a speedtest.

And now that I'm writing it, it's indeed common knowledge just for me; I'm adding it to the comment.